docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md

```diff
@@ -56,135 +56,112 @@ This tutorial assumes that:
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <version>2.1</version>
-        <executions>
-          <execution>
-            <id>copy-dependencies</id>
-            <phase>prepare-package</phase>
-            <goals>
-              <goal>copy-dependencies</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/lib</outputDirectory>
-              <overWriteReleases>true</overWriteReleases>
-              <overWriteSnapshots>true</overWriteSnapshots>
-              <overWriteIfNewer>true</overWriteIfNewer>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
-</project>
-```
-
-### Write the sample Java application
-
-Create the appropriate directory structure as expected by Maven.
-
-```sh
-$ mkdir -p src/main/java/com/yugabyte/sample/apps
-```
-
-Copy the following contents into the file `src/main/java/com/yugabyte/sample/apps/YBSqlHelloWorld.java`.
-
-```java
-package com.yugabyte.sample.apps;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-
-public class YBSqlHelloWorld {
-
-    public static void main(String[] args) throws ClassNotFoundException, SQLException {
+1. Create a project called "MySample".
+
+    ```sh
+    $ mvn archetype:generate \
+        -DgroupId=com.yugabyte \
+        -DartifactId=MySample \
+        -DarchetypeArtifactId=maven-archetype-quickstart \
+        -DinteractiveMode=false
+
+    $ cd MySample
+    ```
+
+1. Replace the contents of the `pom.xml` file with the following:
+
+    ```xml
+    <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+      <modelVersion>4.0.0</modelVersion>
+      <groupId>com.yugabyte</groupId>
+      <artifactId>MySample</artifactId>
+      <packaging>jar</packaging>
+      <version>1.0-SNAPSHOT</version>
+      <name>MySample</name>
+      <url>http://maven.apache.org</url>
+      <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+          <version>3.8.1</version>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.2.14</version>
+        </dependency>
+      </dependencies>
+    </project>
+    ```
+
+1. Install the added dependency.
+
+    ```sh
+    $ mvn install
+    ```
+
+1. Make a new directory for your app.
```

Comment:
You don't this "mkdir" step.
The above step of "mvn install" should have already made the directory "com/yugabyte".

Reply:
removed!